rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/Requester.hpp

```diff
@@ -0,0 +1,126 @@
+#ifndef __rosidl_typesupport_opensplice_cpp__Requester__h__
```

Comment:
Two things:

- Can you put in a copyright/license header here?
- Even though it's currently very inconsistent, I believe we've decided to do all lowercase with underscores for file names, so this would be `requester.hpp` rather than `Requester.hpp` (the include guard should be updated to match). Later I'll make a pass to make everything consistent, but we should try to match the expected style on new stuff.